dev/test/query.ts

```diff
@@ -1331,6 +1425,116 @@ describe('where() interface', () => {
       'Value for argument "opStr" is invalid. Acceptable values are: <, <=, ==, !=, >, >=, array-contains, in, not-in, array-contains-any'
     );
   });
+
+  it('supports composite filters', () => {
```

Comment:
nit: this test does `AND(OR(AND(`. Can you also add another test that does `OR(AND(OR` ? Basically want to have coverage for the top-level filter being an OR.

Reply:
done. Consider doing the same for java-firestore. I believe these were copied direct.

Reply:
Will do. Thanks